examples/jumbo-frame/README.md

```diff
@@ -0,0 +1,101 @@
+# Jumbo Frame Support
```

Comment:
Had some offline discussion that the language is a bit too informal in this readme for public facing AWS stuff, updating in next commit to remove first and second person references, exclamation points, and most adjectives/adverbs.